blogposts/2021/dev-tool-time-liz-fong-jones.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Accessibility, observability, and sustainability: Dev Tool Time with Liz Fong-Jones'
+description: 'Liz Fong-Jones, Principle Developer Advocate at Honeycomb and long-time Site Reliability Engineer (SRE), shares how nano, Honeycomb, and ARM processors fit together into a moveable, minimal workflow for development.'
+author: Scott Bailey
+authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
+publishDate: 2021-09-16T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-liz-fong-jones
+heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/dev-tool-time-fong-jones-cover.jpg
+socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/dev-tool-time-fong-jones-cover.jpg
+published: true
+---
+
+When [Liz Fong-Jones](https://www.lizthegrey.com/), Principal Developer Advocate at Honeycomb and experienced Site Reliability Engineer (SRE), joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on September 1, 2021, she shared how her setup and workflow bridge the current state of development and the future that's already arriving.
+
+You can watch the recording of this episode at the [bottom of this blog post](#Watch-the-recording), but first let's check out Liz's main desk setup:
+
+## Hardware setup
+
+Liz's setup stresses ergonomics, and includes two monitors and a mechnical keyboard. Liz also uses a Blue Yeti mic and Logitech (?) webcam. On her desk are two computers: an AMD64 box and an ARM64 box with a 16 cores at 2.2GHz, 32 GB ram.
```

Comment:
```suggestion
Liz's setup stresses ergonomics, and includes two monitors and a mechnical keyboard. On her desk are two computers: an AMD64 box and an ARM64 box with a 16 cores at 2.2GHz, 32 GB ram.
```

@csbailey5t I forgot to mention this in my review, but I think the ? is there because we aren't sure of the brand of webcam? It looks like an error or a note to ourselves so I'm going to remove this line just in case. We can add it back when we've confirmed.